app/src/ui/changes/changed-file-details.tsx

```diff
@@ -16,12 +18,25 @@ export class ChangedFileDetails extends React.Component<IChangedFileDetailsProps
     const status = this.props.status
     const fileStatus = mapStatus(status)
 
+    let metadataElement: JSX.Element | undefined
+    const diff = this.props.diff
+    if (diff.kind === DiffType.Text) {
+      if (diff.lineEndingsChange) {
+        const message = `Warning: line endings change from '${diff.lineEndingsChange.from}' to '${diff.lineEndingsChange.to}'.`
```

Comment:
I'll try both, but "Warning: line endings **have changed** from .." might be the best fit here.